### PR TITLE
chart: Add extraPaths to Ingress of GitHub Webhook Server

### PR DESCRIPTION
**Pull request by KENNYSOFT**
Hi, congratulation that the team released appVersion 0.21.0 and chartVersion 0.16.0.

I want to add some extra paths to the Ingress of the GitHub Webhook Server but every `paths` is automatically connected to the server, so I've add `extraPaths` just like [grafana](https://github.com/grafana/helm-charts/tree/main/charts/grafana) or [argo-cd](https://github.com/argoproj/argo-helm/tree/master/charts/argo-cd) charts.

References:

* https://github.com/grafana/helm-charts/blob/afcaf811ae181b64032994e677790266d49049db/charts/grafana/values.yaml#L200-L213
* https://github.com/grafana/helm-charts/blob/afcaf811ae181b64032994e677790266d49049db/charts/grafana/templates/ingress.yaml#L40-L42
* https://github.com/argoproj/argo-helm/blob/591de85984906ad5e98e4bb9785448dc4a78eb8a/charts/argo-cd/values.yaml#L1027-L1040
* https://github.com/argoproj/argo-helm/blob/591de85984906ad5e98e4bb9785448dc4a78eb8a/charts/argo-cd/templates/argocd-server/ingress.yaml#L38-L40

### CODE REVIEW

File: charts/actions-runner-controller/templates/githubwebhook.ingress.yaml [hunk 1]
```diff
@@ -36,6 +36,9 @@ spec:
     - host: {{ .host | quote }}
       http:
         paths:
+          {{- if .extraPaths }}
+          {{ toYaml .extraPaths | indent 10 }}
```
Comment (mumoshu) [inline charts/actions-runner-controller/templates/githubwebhook.ingress.yaml:40]: @KENNYSOFT Hey! Just wondering, but doesn't this result in the first extraPaths item to be rendered with 10 + 10 = 20 spaces indentation? Perhaps you meant `nindent 10`, right?
```suggestion
          {{ toYaml .extraPaths | nindent 10 }}
```

Comment (KENNYSOFT) [reply charts/actions-runner-controller/templates/githubwebhook.ingress.yaml:40]: Sorry for the late, just confirmed by using command:

```bash
helm template --debug --set githubWebhookServer.ingress.enabled=true --set githubWebhookServer.ingress.hosts[0].extraPaths.path="/*" .
```

result:

```yaml
# Source: actions-runner-controller/templates/githubwebhook.ingress.yaml
apiVersion: networking.k8s.io/v1
kind: Ingress
metadata:
  name: RELEASE-NAME-actions-runner-controller-github-webhook-server
  labels:
    helm.sh/chart: actions-runner-controller-0.16.0
    app.kubernetes.io/name: actions-runner-controller
    app.kubernetes.io/instance: RELEASE-NAME
    app.kubernetes.io/version: "0.21.0"
    app.kubernetes.io/managed-by: Helm
spec:
  rules:
    - host:
      http:
        paths:
                    path: /*
```

😱

**Comment by KENNYSOFT** [comment]
I just confirmed that applying your suggestion results:

```yaml
# Source: actions-runner-controller/templates/githubwebhook.ingress.yaml
apiVersion: networking.k8s.io/v1
kind: Ingress
metadata:
  name: RELEASE-NAME-actions-runner-controller-github-webhook-server
  labels:
    helm.sh/chart: actions-runner-controller-0.16.0
    app.kubernetes.io/name: actions-runner-controller
    app.kubernetes.io/instance: RELEASE-NAME
    app.kubernetes.io/version: "0.21.0"
    app.kubernetes.io/managed-by: Helm
spec:
  rules:
    - host:
      http:
        paths:

          path: /*
```

(I provided only `path: /*` to see how it templated.) Please re-review! Thanks.

**Comment by mumoshu** [comment]
@KENNYSOFT Hey! Sorry for the delay and thanks for the fix.

Rereviewed this just now, and I added a minor tweak 56e1f5b

That's all and this LGTM! Thanks for your contribution

**Review by mumoshu — APPROVED** [review]

**Pull request merged**